en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Under **Statements**, select **Variable** and enter details as follows:
+
+        | **Field**      | **Value**                                    |
+        |----------------|----------------------------------------------|
+        | **Type**       | `decimal`                                    |
+        | **Name**       | `totalCasesPerMillion`                       |
+        | **Expression** | `<decimal>(totalCases / populationMillions)` |
+
+    3. Click **Save**.
+
+7. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
+
+    1. To build the `json` payload with data of the total cases per million in the population, add a variable.
+
+        Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
+
+         | **Field**      | **Value**                     |
+         |----------------|-------------------------------|
+         | **Type**       | `json`                        |
+         | **Name**       | `payload`                     |
+         | **Expression** | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
+
+    2. Click **Save**.
+
+    3. To respond with the JSON payload, add a `Return` statement.
+
+        Click the last **+** icon in the low-code diagram and click **Return**.
+
+        In the **Return Expression** field, enter `payload`.
+
+        Save the information.
+
+Now you have completed designing the `CovidStatus` service.
+
+The low-code diagram looks as follows:
+
+ ![Add service construct](../assets/img/tutorials/rest-api/low-code-diagram.png){.cInlineImage-full}
+
+The code view looks as follows:
+
+```ballerina
+import ballerinax/worldbank;
+import ballerinax/covid19;
+import ballerina/http;
+
+service /CovidStats on new http:Listener(9090) {
+    resource function get stats/[string country]() returns json|error {
+        covid19:Client covid19Client = check new ({});
+        covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
+        int totalCases = <int>statusByCountry.cases;
+        worldbank:Client worldbankClient = check new ({});
+        worldbank:IndicatorInformation[] populationByCountry = check worldbankClient->getPopulationByCountry(country);
+        int populationMillions = (populationByCountry[0]?.value ?: 0) / 1000000;
+        decimal totalCasesPerMillion = <decimal>(totalCases / populationMillions);
+        json payload = {country: country, totalCasesPerMillion: totalCasesPerMillion};
+        return payload;
+    }
+}
+```
+Now you can run the REST API and test it to see whether it works as expected.
+
+### Run and test the REST API
+
+To run the API, click **Run** (above the low-code diagram).
+
+Once the REST API is successfully started, the following appears in the terminal log.
```

Comment:
What does running a REST API mean? Shall we mention that?

Reply:
How does someone access the terminal log? we haven't mentioned that before.

Reply:
The terminal opens on its own in the VS Code Editor when you click **Run*.*